scripts/run_app.sh

```diff
@@ -44,11 +51,11 @@ if [ ! -s "app/static" ]; then
   npm run compile
 fi
 
-url="`python token_generator.py`"
+url="`run_python token_generator.py`"
 
-if [ -z "${TRAVIS}" ]; then
+if [ -z "${TRAVIS}"] || [-z "${DOCKER}"]; then
   open_url $url &
```

Comment:
Or we could abandon the use of the run_app.sh script when using docker and change the entrypoint to python3 application.py runserver?